src/coreComponents/mesh/generators/InternalWellboreGenerator.cpp

```diff
@@ -346,6 +346,10 @@ void InternalWellboreGenerator::coordinateTransformation( arrayView2d< real64, n
   SortedArray< localIndex > & tnegNodes = nodeSets["tneg"];
   SortedArray< localIndex > & tposNodes = nodeSets["tpos"];
 
+  // Nodesets for the casing-cement and cement-rock interfaces of a cased wellbore
+  SortedArray< localIndex > & rinterface1Nodes = nodeSets["rIT1"];
+  SortedArray< localIndex > & rinterface2Nodes = nodeSets["rIT2"];
```

Comment:
Doing so you're adding a component to the public interface of GEOSX.
I think it should be document somewhere, otherwise you'll be the only one to be able to use the feature.

Changing the names in the future will result in a modification of all the input files.
It's something we'll be in trouble to fix right now.
So maybe could we use a more explicit name (`rIT1` and `rIT2` are complicated to  understand) so we hopefully do not have to change too often 🤣 .

Reply:
@TotoGaz Thank you very much! Following your suggestion, a doc is added for the nodesets generated by the `InternalMesh` generator and the `InternalWellbore` generator. Also, more explicit `setNames` are now used.